.gn

```diff
@@ -167,8 +167,8 @@ exec_script_whitelist =
       "//tools/gritsettings/BUILD.gn",
     ]
 
-import("//chrobalt/build/configs/chrobalt.gni")
-if (chrobalt) {
+import("//cobalt/build/configs/cobalt.gni")
+if (cobalt) {
```

Comment:
Could you actually rename this to is_cobalt to align with the gn style guide? Wasn't addressed in https://github.com/youtube/cobalt/pull/4215#discussion_r1793559912

Reply:
done

Reply:
Thanks!